pkg/scaffold/api.go

```diff
@@ -260,13 +271,30 @@ func (api *API) scaffoldV2() error {
 	return nil
 }
 
+// isNewGroup will return true if the group informed is diff of the pre-existents
+func (api *API) isNewGroup(r *resource.Resource) bool {
+	for _, existingGroup := range api.project.ResourceGroups() {
+		if strings.ToLower(r.Group) != strings.ToLower(existingGroup) {
+			return true
+		}
+	}
+	return false
+}
+
 // Since we support single group only in v2 scaffolding, validate if resource
```

Comment:
Assume we want to remove this comment before this is merged?

Reply:
I will remove it.